popup/css/bookmarkManager.css

```
@@ -0,0 +1,1739 @@
+  font-size: 13px;
+}
+
+.error-overlay {
+  position: fixed;
+  inset: 0;
+  background: rgba(255, 255, 255, 0.98);
+  color: #cc0000;
+  font-size: 14px;
+  padding: 12px;
+  overflow-y: auto;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)